From 0369a8c545254a49d68cdeb11a1b9f8a3ea34e23 Mon Sep 17 00:00:00 2001
From: "Dr. Greg" <greg@enjellic.com>
Date: Sat, 6 Feb 2021 11:05:02 -0600
Subject: [PATCH 13/50] Rename structure defining an event point.

This update converts the structure used to describe each point on
the event surface from contour to event_point.
---
 security/integrity/ima/ima_events.c | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/security/integrity/ima/ima_events.c b/security/integrity/ima/ima_events.c
index ee34ced3b805..98c9acd17329 100644
--- a/security/integrity/ima/ima_events.c
+++ b/security/integrity/ima/ima_events.c
@@ -126,8 +126,8 @@ struct AIevent AI_events[] = {
 };
 
 
-/* Structure defining an event contour point. */
-struct contour {
+/* Structure defining an event point. */
+struct event_point {
 	struct list_head list;
 	char point[WP256_DIGEST_SIZE];
 	bool valid;
@@ -174,7 +174,7 @@ struct ima_events_map {
 	char hostid[WP256_DIGEST_SIZE];
 	char measurement [WP256_DIGEST_SIZE];
 
-	struct contour cp;
+	struct event_point cp;
 	unsigned int contour_count;
 	struct mutex contour_list_mutex;
 	struct list_head contour_list;
@@ -312,7 +312,7 @@ static void get_identity(struct actor_identity *id)
 static int have_contour(char *point)
 {
 	int retn = 0;
-	struct contour *entry;
+	struct event_point *entry;
 	struct ima_events_map *map = current_map();
 
 	mutex_lock(&map->contour_list_mutex);
@@ -340,7 +340,7 @@ static int have_contour(char *point)
 static int have_valid_contour(char *point)
 {
 	int retn = 0;
-	struct contour *entry;
+	struct event_point *entry;
 	struct ima_events_map *map = current_map();
 
 	mutex_lock(&map->contour_list_mutex);
@@ -368,7 +368,7 @@ static int have_valid_contour(char *point)
 static int add_contour_point(char *point, bool valid)
 {
 	int retn = 1;
-	struct contour *entry;
+	struct event_point *entry;
 	struct ima_events_map *map = current_map();
 
 	entry = kzalloc(sizeof(*entry), GFP_KERNEL);
@@ -921,9 +921,9 @@ static void ima_contours_stop(struct seq_file *c, void *pos)
 
 static int ima_contours_show(struct seq_file *c, void *identity)
 {
-	struct contour *id;
+	struct event_point *id;
 
-	id = list_entry(identity, struct contour, list);
+	id = list_entry(identity, struct event_point, list);
 	seq_printf(c, "%*phN\n", WP256_DIGEST_SIZE, id->point);
 	return 0;
 }
@@ -1772,7 +1772,7 @@ void ima_free_ns(struct kref *kref)
 {
 	unsigned int cnt;
 	struct ima_events_namespace *ns;
-	struct contour *centry, *tmp_centry;
+	struct event_point *centry, *tmp_centry;
 	struct trajectory *tentry, *tmp_tentry;
 
 	ns = container_of(kref, struct ima_events_namespace, kref);
-- 
2.31.1

